lib/record/absoluteUri.dart

```diff
@@ -36,10 +32,12 @@ class AbsoluteUriRecord extends NDEFRecord {
 
   //absoluteURI record has no payload
   Uint8List? get payload {
-    return null;
+    return Uint8List(0);
```

Comment:
This might be expensive to create `Uint8List` on the fly, as it depends on external factory which always create an `ByteBuffer`.